apps/Kernel/Makefile

```diff
@@ -47,8 +47,8 @@ PFLAGS += -DBLIP_SEND_ROUTER_ADVERTISEMENTS=0
 
 # Configure the number of times BLIP tries to send a packet and how long it
 # waits between attempts
-PFLAGS += -DBLIP_L2_RETRIES=6
-PFLAGS += -DBLIP_L2_DELAY=40
+PFLAGS += -DBLIP_L2_RETRIES=11
+PFLAGS += -DBLIP_L2_DELAY=2
```

Comment:
Yes, but that was for the experiment where I still had hardware retries. When I implemented Software CSMA and removed hardware retries, I halved the delay between retries so that the average delay between retries is the same. I describe this briefly in Section 10.